wagtail/admin/templates/wagtailadmin/pages/listing/_pagination.html

```diff
@@ -11,12 +11,18 @@
     <ul>
         <li class="prev">
             {% if page.has_previous %}
-                <a data-page="{{ page.previous_page_number }}" href="{{ base_url }}{% pagination_querystring page.previous_page_number page_key=page_key %}" class="icon icon-arrow-left {{ classnames }}">{% trans "Previous" %}</a>
+                <a data-page="{{ page.previous_page_number }}" href="{{ base_url }}{% pagination_querystring page.previous_page_number page_key=page_key %}" class="{{ classnames }}">
+                    {% icon name="arrow-left" class_name="default" %}
```

Comment:
I’m not a big fan of this `default` class name (default what?), but this was here before so oh well.